src/theme/token-details-widget-theme.ts

```diff
@@ -0,0 +1,34 @@
+import { colors } from './colors'
+
+export const LIGHT_THEME = {
```

Comment:
I think if you do `useTheme`, might get the dark/light theme switching for free?

Reply:
That was my first thought too, but `useTheme` gives you the current context value for the `styled-components` theme, and we need the (significantly different) widget theme here only.

Reply:
ah okay, np

Reply:
similar comment around whether this is token-details specific as below